mdx2/command_line/report.py

```
@@ -0,0 +1,264 @@
+    # Input nexus files. Will be inspected for object sources, and passed to the template
+    input_files: list[str] = field(positional=True, nargs="*", default_factory=list)
+
+    template: ExecutableNotebook = subgroups(
+        {
+            p._template_name: p
+            for p in [_ExampleParameters, ScalingModelParameters, VisualizationParameters, MapStatisticsParameters]
+        },
+        default="visualization",
+    )
+
+    metadata: Metadata = Metadata()  # metadata fields that can be overridden on the CLI
```

Ruff (0.15.10)

[warning] Do not perform function call `Metadata` in dataclass defaults

(RUF009)